config-example.yaml

```diff
@@ -31,6 +31,8 @@ keep_alive_monitor:
 daily_stats:
   enable: true
   time_of_day: 21
+  frequency_hours: 24
+  debug_mode: false # Send notifications every 5 minutes
```

Comment:
I'm not a fan of this implementation, since in 99% of use cases you will only run this once and then have to change the config and restart chiadog. Furthermore, it feels wrong to add a debug mode specifically for the `daily_stats` config. 

I would prefer to add this as an optional argument to the argument parser that sends a one-time notification instead. There's also no use sending the daily_stats specifically, as they will only contain 5 minutes worth of data.

Reply:
Agree, will remove.